packages/react-native-app-auth/react-native-app-auth.podspec

```diff
@@ -9,7 +9,8 @@ Pod::Spec.new do |s|
   s.license      = package['license']
   s.authors      = package['author']
   s.homepage     = package['homepage']
-  s.platform     = :ios, '10.0'
+  s.ios.deployment_target = '10.0'
+  s.osx.deployment_target = '10.15'
```

Comment:
Technically, AppAuth [lets us](https://github.com/openid/AppAuth-iOS/blob/c89ed571ae140f8eb1142735e6e23d7bb8c34cb2/AppAuth.podspec#L34-L35) deploy down to the following versions:

```ruby
s.ios.deployment_target = "9.0"
s.osx.deployment_target = "10.12"
```

I've chosen `10.15` for this library just to align with the `react-native-macos` template's minimum deployment target.